packages/backend/src/clients/Aws/S3CacheClient.ts

```diff
@@ -79,8 +79,10 @@ export class S3CacheClient {
                     Metadata: metadata,
                 });
                 const response = await this.s3.send(command);
-            } catch (error) {
-                Logger.error(`Failed to upload results to s3. ${error}`);
+            } catch (error: unknown) {
+                Logger.error(
+                    `Failed to upload results to s3. ${String(error)}`,
```

Comment:
instead of doing String(error) , do `getErrorMessage(error)` and import from common if needed